testdata/golang/light/jacobiSymbol.go

```diff
@@ -0,0 +1,29 @@
+package light
+
+import "log"
+
+func jacobiSymbol(a, n uint64) int {
+	if n%2 == 0 {
+		log.Fatal("'n' must be a positive odd integer")
```

Comment:
should we panic here? or return an error?

Reply:
Changed to returning a error